docs/Support.md

```diff
@@ -0,0 +1,50 @@
+# Support
+
+## I need help right now
+
+1. Don't panic
+2. Email emergency@mozillafoundation.org with a description of the problem and a method to contact you
+  - Try to be specific about the problem you're seeing
+  - How can on-call contact you to acknowledge your problem?
+3. Join #fundraising on irc.mozilla.org if you're able
+4. Try to gather additional information about the issue while waiting for on-call to acknowledge
+  - Can you take a screenshot of the problem you're seeing?
+  - Does the problem occur in different browsers? How about on different devices?
+  - Are you encountering any errors with other sites?
+  - Could you try another credit card? Can you sign-in with a different PayPal account?
+  - Are there any error messages appearing in your [browsers web console](https://developer.mozilla.org/en/docs/Tools/Web_Console)?
+5. If on-call has not acknowledged your issue in 15 minutes then use the Mozilla Phonebook to look up and call the following people
+  - Jon Buckley
+  - Ali Al Dallal
+  - Scott Downe
+  - Simon Wex
+
+## Service-specific support
+
+### AWS
+
+- We have [AWS Business Support](https://aws.amazon.com/premiumsupport/).
+- You can file a support ticket by visiting the [Support Center](https://console.aws.amazon.com/support/home).
+- You must be able to sign-in to the mofo-secure AWS account before you can file a support ticket.
+- You can see the current AWS status by visiting their [status page](http://status.aws.amazon.com/).
+
+### GitHub
+
+- We have no support agreement with GitHub.
+- You can file a support ticket by visiting the [support page](https://github.com/contact).
+- You must have a GitHub account to file a ticket.
+- You can see the current GitHub status by visiting their [status page](https://status.github.com/).
+
+### Heroku
+
+- We have [Heroku Premium Support](https://www.heroku.com/critical).
+- You can file a support ticket by visiting [Heroku Help](https://help.heroku.com/tickets/new?query=help+redirect+premium+support).
+- You must have a Heroku account before you can file an issue.
+- To receive premium support you must sign-up with your @mozilla.org or @mozillafoundation.org email address.
+- You can see the current Heroku status by visiting [status page](https://status.heroku.com/).
+
+### Mozilla
+
+- For non-critical issues you should file an issue in [GitHub](https://github.com/mozilla/donate.mozilla.org/issues) or visit #fundraising on irc.mozilla.org.
+- For critical issues you should page MoFo on-call support with a clear description of the problem and a method to contact you.
+- You can also join #moc on irc.mozilla.org and ask for help from on-call
```

Comment:
The IRC channel for the [https://mana.mozilla.org/wiki/display/MOC/About+the+MOC](Mozilla Operations Center)